readme.md

```diff
@@ -42,6 +42,18 @@ ignore             do not copy files into App whose filenames regex .match this
 prune              runs `npm prune --production` on the app
 asar               packages the source code within your app into an archive
 sign               should contain the identity to be used when running `codesign` (OS X only)
+version-string     This is a hash which holds the verision details of the generated exe (windows only).
+                   These are the following keys it supports
```

Comment:
Yep @malept your message is more appropriate.  And i think `-` is suitable for list. 

Also i can make a pull to https://github.com/sindresorhus/grunt-electron on how to specify the hash. That will be easier